force-app/main/default/classes/CON_ContactMerge_CTRL.cls

```diff
@@ -720,6 +713,17 @@ public with sharing class CON_ContactMerge_CTRL {
 
     }
 
+    private void buildSortedCustomFieldList(Map<String, DescribeFieldResult> customFieldMap, String undeliverableFieldName, List<String> customFieldList) {
```

Comment:
I think a better signature for this would be `private static List<String> buildSortedCustomFieldList(Map<String, DescribeFieldResult> customFieldMap)`. Then on line 629 we could have:

`List<String> customFieldList = buildSortedCustomFieldList(customFieldMap);`